gef-ui/server/controllers/about-facility/index.js

```diff
@@ -50,28 +50,45 @@ const validateAboutFacility = async (req, res) => {
   const coverStartDateDay = body['cover-start-date-day'];
   const coverStartDateMonth = body['cover-start-date-month'];
   const coverStartDateYear = body['cover-start-date-year'];
+  const coverStartDateIsFullyComplete = coverStartDateDay && coverStartDateMonth && coverStartDateYear;
```

Comment:
Is coverStartDateMonth zero indexed i.e. Does Jan = 0? Will coverStartDateIsFullyComplete fail if coverStartDateMonth = 0?

Reply:
Actually, scrap that. I can see it isn't as you use coverEndDateMonth - 1 to set a date object further down.